README.md

```diff
@@ -5,11 +5,1119 @@
+input UserFilter {
+  every: UserWhereInput
+  none: UserWhereInput
+  some: UserWhereInput
+}
+
+input UserOrderByInput {
+  handle: OrderByArg
+  id: OrderByArg
+}
+
+input UserWhereInput {
+  AND: [UserWhereInput!]
+  handle: StringFilter
+  id: StringFilter
+  NOT: [UserWhereInput!]
+  OR: [UserWhereInput!]
+  posts: PostFilter
+}
+
+input UserWhereUniqueInput {
+  id: ID
+}
+```
+
+</details>
+
+<br>
+
+You can find a runnable version of this and other examples at [prisma-labs/nexus-examples](/TODO).
+
+## Reference
+
+### `t.model`
+
+`t.model` is for projecting your models’ fields onto your `Object`s. It is only available inside `Object` definitions. `t.model` properties are called "Field Projectors" and there is one per field on your respective model. They accept configuration allowing you to control how the model's fields are seen through your GraphQL schema.
+
+#### Field Projectors
+
+`t.model` will have Field Projectors for the model whose name matches that of the `Object`. If the `Object` is of a name that does not match any of your models then `t.model` becomes a function allowing you to specify the mapping.
+
+```ts
+// Project User model fields `id` & `name` onto User Object
+
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.name()
+  },
+})
+```
+
+```gql
+type User {
+  name: String
+}
+```
+
+```ts
+// Project User model fields `id` & `name` onto Person Object
+
+objectType({
+  name: 'Person',
+  definition(t) {
+    t.model('User').name()
+  },
+})
+```
+
+```gql
+type Person {
+  name: String
+}
+```
+
+#### `alias` Option
+
+Use `alias` to change the name of the field projected onto the `Object`.
+
+```ts
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.name({ alias: 'handle' })
+  },
+})
+```
+
+```gql
+type User {
+  handle: String
+}
+```
+
+#### `type` Option
+
+Only available for relational model fields. Use `type` to change the name of the projected field type. This is useful when the model that the field relates to has been projected onto a differently named `Object`.
+
+```ts
+objectType({
+  name: 'Article',
+  definition(t) {
+    t.model('Post').title()
+  },
+})
+
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.posts({ alias: 'articles', type: 'Article' })
+  },
+})
+```
+
+```gql
+type Article {
+  title: String
+}
+
+type User {
+  articles: [Article]
+}
+```
+
+#### `ordering` `pagination` `filtering` Options
+
+Only available for list type model fields. Please refer to TODO for details.
+
+### `t.crud`
+
+`t.crud` is for exposing operations against your projected models. It is only available inside `Query` and `Mutation` definitions. `t.crud` properties are called "Operation Publishers". They accept configuration allowing you to control how the operation is seen through your GraphQL schema.
+
+`t.crud` reflects a subset of [Photon](https://photonjs.prisma.io)'s capabilities. For every one model in your Prisma schema there are ten corresponding Operation Publishers.
```

Comment:
Something I picked up from my uni days, first link I found in Google

> A simple rule for using numbers in writing is that small numbers ranging from one to ten (or one to nine, depending on the style guide) should generally be spelled out. Larger numbers (i.e., above ten) are written as numerals.

–– https://www.scribendi.com/advice/when_to_spell_out_numbers_in_writing.en.html

I feel like the adjective "exactly" doesn't add much value, I'll just try writing the numbers out as digits.

Reply:
I actually also prefer spelling out numbers instead of using digits (even for numbers up to hundred), so you don't have to switch to digits for me. Thinking more about this, maybe the "exactly" isn't too important,... IMO it would add _a bit_ of value since it points out that this number is very intentional at this point, it's hard to describe for me but otherwise reading this sentence the ten feels a bit random/lost/out-of-place 🙈 no strong opinion though, so feel free to leave it as is!